mmv1/third_party/terraform/services/compute/data_source_google_compute_networks.go

```diff
@@ -0,0 +1,74 @@
+package compute
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func DataSourceGoogleComputeNetworks() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGoogleComputeNetworksRead,
+
+		Schema: map[string]*schema.Schema{
+
+			"self_link": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"project": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"networks": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+		},
+	}
+}
+
+func dataSourceGoogleComputeNetworksRead(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*transport_tpg.Config)
+	userAgent, err := tpgresource.GenerateUserAgentString(d, config.UserAgent)
+	if err != nil {
+		return err
+	}
+
+	project, err := tpgresource.GetProject(d, config)
+	if err != nil {
+		return err
+	}
+
+	networkList, err := config.NewComputeClient(userAgent).Networks.List(project).Do()
+	if err != nil {
+		return transport_tpg.HandleNotFoundError(err, d, fmt.Sprintf("Network Not Found : %s", project))
+	}
+
+	var networks = make([]string, len(networkList.Items))
+
+	for i := 0; i < len(networkList.Items); i++ {
+		fmt.Println(networkList.Items[i])
```

Comment:
Is this line for debugging?

Reply:
Yes I will remove this line and resubmit it.